covid_xprize/examples/prescriptors/neat/utils.py

```diff
@@ -137,7 +145,4 @@ def get_predictions(start_date_str, end_date_str, pres_df, countries=None):
     # Load predictions to return
```

Comment:
The file `TMP_PRED_FILE_NAME` is created above. Will it still exist when this read happens? Because it was never opened until line 146, so thus it was never closed?